scripts/setup-test-db.js

```diff
@@ -50,8 +50,12 @@ const TEST_USERS = [
   }
 ];
 
+const migrationDir = path.join(__dirname, '..', 'migrations');
 // Path to the SQL migration file
-const migrationFilePath = path.join(__dirname, '..', 'migrations', '001_init.sql');
+const migrationFilePaths = [
+  path.join(migrationDir, '001_init.sql'),
+  path.join(migrationDir, '002_api_keys.sql'),
+];
```

Comment:
same here, ideally we never have to edit this file when we create new migrations